content/docs/command-reference/list.md

```diff
@@ -122,3 +122,25 @@ images/dvc-logo-outlines.png.dvc
 images/owl_sticker.png
 ...
 ```
+
+## Example: Archive project data
+
+An archive is a single file that contains multiple files from a project. It can
+be used to backup project data. We can use `dvc list` to create an archive of
+files and directories (<abbr>outputs</abbr>) tracked by DVC:
```

Comment:
We have to think through the motivation and use cases here. Are we doing what the issue originally suggested? Or one of the options in https://github.com/iterative/dvc.org/issues/1521#issuecomment-558726115? Or something else? And why

p.s. is this ready for review? I see the OR is in draft state.

Reply:
p.s. is this ready for review? I see the PR is in draft state.